psi4/src/psi4/dfmp2/mp2.cc

```diff
@@ -2411,16 +2418,18 @@ void RDFMP2::form_Z()
         Dtemp->scale(0.5);
 
         // Add in the reference contribution
+
         for (int i = 0; i < nocc; ++i)
             Dtemp->add(i, i, 1.0);
-        Ca_ = SharedMatrix(new Matrix("DF-MP2 (unrelaxed) Natural Orbitals", nsopi_, nmopi_));
-        epsilon_a_ = SharedVector(new Vector("DF-MP2 (unrelaxed) NO Occupations", nmopi_));
+        Ca__ = SharedMatrix(new Matrix("DF-MP2 (unrelaxed) Natural Orbitals", nsopi_, nmopi_));
+        epsilon_a__ = SharedVector(new Vector("DF-MP2 (unrelaxed) NO Occupations", nmopi_));
```

Comment:
I don't think we want to retain `Ca_` and `epsilon_a_` as the original SCF values, its ok if Wavefunctions update themselves. For example, the MCSCF Wavefunction rotates its own orbitals.

These values will not get used within the MP2 code, but are useful for things like:
```
set mp2_opdm true # or whatever
mp2_grad, mp2_wfn = psi4.energy("MP2")
psi4.energy("OCCSD", ref_wfn = mp2_wfn)
```

The relaxed densities are odd because they rotate the `occupied x virtual` parts of the OPDM while the unrelaxed density only rotates the `occupied x occupied` and `virtual x virtual` areas. The former actually changes the Wavefunction while the latter helps compress the information. (Hopefully I didn't screw this up, its been awhile since I have done MP2 densities)